src/index.html

```diff
@@ -6,7 +6,420 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="header__logo">
+        <a
+          href="#"
+          class="header__link"
+        >
+          <img
+            src="images/logo.png"
+            alt="#"
+            class="header__picture"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">
+                 Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                 Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Смартфоны и телефоны
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+              Гаджеты
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Планшеты
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Фото
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Видео
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__row">
+          <div class="card" data-qa="card">
+            <div class="card__picture">
```

Comment:
In the next project better use `mix` and not so much classes
```suggestion
    <main>
      <div class="catalog">
          <div class="card catalog__card(renaming for row)" data-qa="card">
            <div class="card__picture">
```

Reply:
++